FAQ/en.md

```diff
@@ -0,0 +1,61 @@
+# Frequently Asked Questions
+
+## General
+
+### What state is Quaver currently in?
+
+Quaver is currently in the closed Alpha state. In order to use online features of Quaver, you need to sign up for the Alpha.
+
+### Where can I sign up for the Quaver Alpha?
+
+We announce sign-ups before they open, but until then, you will have to wait. Sign-ups are being held during specific time periods, and are not open at all time.
+
+### Can I play without being part of the Quaver Alpha?
+
+Yes, you can download an offline version from Quaver’s [Github page](https://github.com/Quaver/Quaver/releases), However, you won’t be able to experience any of the online features.
+
+### Does this mean I can’t download maps from the Quaver website?
+
+That is correct, however, Adri (a member of the Quaver team) offers an alternate download [here](https://rhythmgamers.net/pack/) in the form of map packs for anyone to access. Keep in mind he doesn’t update those packs very frequently.
+
+### Is there an estimate for the public release date?
+
+We do not have an estimate for the public release date yet.
+
+
+## Gameplay
+
+### What key modes does Quaver support?
+
+Quaver currently only supports 4 keys and 7 keys.
+
+### Can I import maps from other games?
+
+Yes, Quaver currently supports .osu and .sm files and we are working on supporting more file formats.
```

Comment:
`Yes, Quaver currently supports .osz and .sm files. To import them, drag the files into the game window.`.

Reply:
Applied